### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by max-zaytsev**
Update dependencies to resolve vulnerability issues.
Bumped up "azure-devops-node-api" from "^8.1.1" to "^10.2.2"

List of commands tested locally:

- login
- workitem create
- workitem query
- workitem update 
- workitem show
- build list 
- build queue
- build show
- build tasks create
- build tasks upload
- build tasks list
- build tasks delete
- extension create
- extension create
- extension init
- extension isvalid
- extension publish
- extension show
- extension share
- extension unpublish
- extension unshare
- extension resources

Related issue: https://github.com/microsoft/tfs-cli/issues/380

- [x]  There are no risky dependency updates
- [x]  Changes have been tested
- [ ]  Enough test coverage for changes and current test coverage for the task doesn't look poor
There are no tests yet. Tested manually
- [x]  We understand how tasks are working, how changes affect task behavior
- [x]  There are no breaking changes
- [x]  There are no other concerns
- [ ]  I have not discovered any new uncovered test/use cases
There are no tests yet. Tested manually

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -1,15 +1,21 @@
 {
 	// See https://go.microsoft.com/fwlink/?LinkId=733558
 	// for the documentation about the tasks.json format
-	"version": "0.1.0",
+	"version": "2.0.0",
```
Comment (anatolybolshakov) [inline .vscode/tasks.json:4]: Do we need changes in .vscode folder?

**Review by anatolybolshakov — APPROVED** [review]
LGTM, thanks! Please take a look at comments also

**Comment by AndreyIvanov42** [comment]
Manually tested. Command functionality is not suffered

**Review by AndreyIvanov42 — APPROVED** [review]

**Comment by anatolybolshakov** [comment]
@alexander-smolyakov could you please take a look?

**Review by alexander-smolyakov — APPROVED** [review]

**Pull request merged**